tools/kmt/kmt.c

```diff
@@ -1719,17 +1717,15 @@ km_rec_print_cmn(struct km_inst *inst, struct km_rec *r, const char *fmt, hse_er
 
     dst = vbuf;
     for (i = 0; i < 32 && i < r->vlen; ++i) {
-        const char tab[] = {
-            '0', '1', '2', '3', '4', '5', '6', '7',
-            '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'
-        };
+        const char tab[] = { // eol
+            '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f' };
```

Comment:
Better the way it was, easier to find the index you want, and doesn't leave the closing brace way off to the right where you don't expect it...

Reply:
Prob should protect the big array in bloom filter..

Reply:
fixed in recent push

Reply:
Easier to verify the other way, plus I hate the assymetry and inconsistency of the brace placement...  Don't worry, I'll "fix" it next time I'm there..